src/convert.jl

```diff
@@ -218,6 +218,31 @@ function convert(::Type{HPolytope}, P::VPolytope)
     return tohrep(P)
 end
 
+"""
+    convert(::Type{VPolytope}, X::LazySet; [prune]::Bool=true)
+
+Generic conversion to polytope in vertex representation.
+
+### Input
+
+- `type`  -- target type
+- `X`     -- set
+- `prune` -- (optional, default: `true`)
+
+### Output
+
+The given set represented as a polytope in vertex representation.
+
+### Algorithm
+
+We compute the list of vertices of `X` and wrap the result in a polytope in
+vertex representation, `VPolytope`. Use the option `prune` to select whether or not
+to apply a convex hull previous to the construction of the polytope.
```

Comment:
```suggestion
to remove redundant vertices previous to the construction of the polytope.
```
"convex hull" is an implementation detail and the effect is not necessarily clear.